app/interactors/create_order.rb

```diff
@@ -59,13 +53,11 @@ def create_order_from_cart(params, cart, user)
 
     order.apply_delivery_address(cart.delivery_location)
 
-    DeliveryNote.where(cart_id:cart.id).each do |dn|
-      #binding.pry # there is no id available yet, it's not created
-      dn.update_attributes(order_id:order.id)
-    end
-
```

Comment:
So this code never did anything actually? And was duplicated further down?

Reply:
It looks like it was mistakenly added in two places as far as I can tell.